fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1542,6 +1680,49 @@ ACTOR Future<Void> masterProxyServerCore(
 			rep.version = commitData.committedVersion.get();
 			req.reply.send(rep);
 		}
+		when(ExecRequest _execReq = waitNext(proxy.execReq.getFuture())) {
```

Comment:
I think i tried the path of putting it in cluster controller and ran into issues, I think the interface exposed by CC to the client is limited or something like that.

Reply:
In FDB 6 `ClientWorkerInterface` is exposed. So you should probably put it there and call it directly from the client instead of going through the proxy.